packages/flutter/lib/src/cupertino/icons.dart

```diff
@@ -47,15 +47,15 @@ class CupertinoIcons {
   // We need to leave them as-is with the same codepoints for backward
   // compatibility with cupertino_icons <0.1.3.
 
-  /// A thin left chevron.
+  /// <i class='cupertino-icons md-36'>chevron_left</i> &#x2014; Cupertino icon for a thin left chevron.
```

Comment:
Thanks for typing these in!

Reply:
Picked up some regex skills to avoid doing this too manually!